packages/nodes-base/nodes/EmailReadImap/EmailReadImap.node.ts

```diff
@@ -399,7 +429,7 @@ export class EmailReadImap implements INodeType {
 							}
 						}
 						if (staticData.lastMessageUid !== undefined) {
-							searchCriteria.push(['UID', `${staticData.lastMessageUid as number}:*`]);
+							searchCriteria.push(['UID', `${staticData.lastMessageUid as number + 1}:*`]);
```

Comment:
comment just below this line
`
							 * - You cannot filter using ['UID', 'CURRENT ID + 1:*'] because IMAP
							 * won't return correct results if current id + 1 does not yet exist.
`